pylab/website/urls.py

```diff
@@ -11,6 +11,8 @@
     url(r'^projects/create/$', views.project_create, name='create-project'),
     url(r'^projects/(?P<project_slug>%s)/$' % slug, views.project_details, name='project-details'),
     url(r'^projects/(?P<project_slug>%s)/update/$' % slug, views.project_update, name='project-update'),
+    url(r'^events/(?P<year>[0-9]{4})/(?P<month>[0-9]{2})/(?P<day>[0-9]+)/(?P<event_slug>%s)/$' % slug,
+        views.event_details, name='event-details'),
```

Comment:
Since we are going to add more urls on top of base event details url, this can be refactored into something smaller and reusable, like in my branch: https://github.com/python-dirbtuves/website/blob/monday-event/pylab/website/urls.py#L15